### PR TITLE
Update Community Docs

### PR DESCRIPTION
**Pull request by adam-palazzo**
This is mainly license updates, but there are a couple of quick styling edits too

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,8 +28,9 @@ We would be happy to accept PRs! If you want to work on something, it'd be good
 
 We also have **regular community calls**, which we announce in the issues in [the @orbitdb welcome repository](https://github.com/orbitdb/welcome/issues). Join us!
 
-For specific guidelines for contributing to this repository, check out the [Contributing guide](CONTRIBUTING.md). For more on contributing to OrbitDB in general, take a look at the [orbitdb welcome repository](https://github.com/orbitdb/welcome). Please note that all interactions in @orbitdb fall under our [Code of Conduct](CODE_OF_CONDUCT.md).
+For specific guidelines for contributing to this repository, check out the [Contributing guide](CONTRIBUTING.md). For more on contributing to OrbitDB in general, take a look at the [orbitdb welcome repository](https://github.com/orbitdb/welcome). Please note that all interactions in [@orbitdb](https://github.com/orbitdb) fall under our [Code of Conduct](CODE_OF_CONDUCT.md).
```
Comment (RichardLitt) [inline README.md]: Change to `[@OrbitDB]`

Comment (RichardLitt) [reply README.md]: Both times.

File: README.md [hunk 2]
```diff
@@ -28,8 +28,9 @@ We would be happy to accept PRs! If you want to work on something, it'd be good
...
 
 ## License
 
-[MIT](LICENSE) © 2016 Haad, 2019 Haja Networks Oy
+[MIT](LICENSE) © 2016-2018 Haad,
```
Comment (RichardLitt) [inline README.md]: Remove comma? User: "2016-2018 Haad, 2019 Haja Newtworks Oy".

**Review by RichardLitt — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -26,10 +26,11 @@ The methods for this module are not yet documented. If you can, please help us b
...
+We also have **regular community calls**, which we announce in the issues in [the @OrbitDB welcome repository](https://github.com/orbitdb/welcome/issues). Join us!
 
-For specific guidelines for contributing to this repository, check out the [Contributing guide](CONTRIBUTING.md). For more on contributing to OrbitDB in general, take a look at the [orbitdb welcome repository](https://github.com/orbitdb/welcome). Please note that all interactions in @orbitdb fall under our [Code of Conduct](CODE_OF_CONDUCT.md).
+For specific guidelines for contributing to this repository, check out the [Contributing guide](CONTRIBUTING.md). For more on contributing to OrbitDB in general, take a look at the [OrbitDB welcome repository](https://github.com/orbitdb/welcome). Please note that all interactions in [@OrbitDB](https://github.com/orbitdb) fall under our [Code of Conduct](CODE_OF_CONDUCT.md).
 
 ## License
 
-[MIT](LICENSE) © 2016 Haad, 2019 Haja Networks Oy
+[MIT](LICENSE) © 2016-2018 Haad
+[MIT](LICENSE) © 2019 Haja Networks Oy
```
Comment (RichardLitt) [inline README.md]: This will show up as one line in Markdown. Switch to: 

```
[MIT](LICENSE) © 2016-2018 Haad, 2019 Haja Networks Oy
```

**Review by RichardLitt — CHANGES_REQUESTED** [review]
Close!

**Comment by RichardLitt** [comment]
Thank you!

File: LICENSE [hunk 4]
```diff
@@ -1,6 +1,8 @@
 The MIT License (MIT)
 
-Copyright (c) 2016 Haad
+Copyright (c) 2016-2018 Haad
+Copyright (c) 2019 Haja Networks Oy
+
```
Comment (haadcode) [inline LICENSE:5]: Unfortunately, these are not correct as this was done under Protocol Labs, so it's:
(c) 2016-2017 Protocol Labs

I believe Haja Networks hasn't made a significant contribution to this to claim on copyright, so I believe it's only that ^. 

cc @RichardLitt

Comment (RichardLitt) [reply LICENSE:5]: Thanks, @haadcode. Changing.

**Pull request merged**